src/Be.Vlaanderen.Basisregisters.ProjectionHandling.Runner/RunnerDbContext.cs

```diff
@@ -8,13 +8,15 @@ namespace Be.Vlaanderen.Basisregisters.ProjectionHandling.Runner
     using Microsoft.EntityFrameworkCore;
     using ProjectionStates;
 
-    public abstract class RunnerDbContext<TContext> : DbContext where TContext : DbContext
+    public abstract class RunnerDbContext<TContext> : DbContext
+        where TContext : DbContext
     {
         public abstract string ProjectionStateSchema { get; }
 
-        public DbSet<ProjectionStateItem> ProjectionStates { get; set; }
+        public DbSet<ProjectionStateItem>? ProjectionStates { get; set; }
```

Comment:
By doing this, we create different problems see line 70 in `Runner.cs`.
Can we not make this nullable?

https://learn.microsoft.com/en-us/ef/core/miscellaneous/nullable-reference-types see DbContext and DbSet

Reply:
i assume you meant not nullable -- fixed